spec/factories/orders.rb

```diff
@@ -0,0 +1,27 @@
+FactoryBot.modify do
+  factory :order do
+    trait :with_line_item do
+      transient do
+        variant { FactoryGirl.create(:variant) }
+      end
+
+      after(:create) do |order, evaluator|
+        create(:line_item, order: order, variant: evaluator.variant)
+      end
+    end
+
+    trait :completed do
```

Comment:
Spree has the `completed_order_with_totals` factory, which we reused to build more complex ones in OFN. Can't we use just that one?